cleverpush/src/main/java/com/cleverpush/stories/models/Preview.java

```diff
@@ -0,0 +1,89 @@
+
+package com.cleverpush.stories.models;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+import java.io.Serializable;
+
+public class Preview implements Serializable {
```

Comment:
SerialVersionUID missing, when implementing Serializable

https://stackoverflow.com/questions/285793/what-is-a-serialversionuid-and-why-should-i-use-it

Reply:
also if youre working with json you dont need serializable